core/core/urls.py

```diff
@@ -15,11 +15,29 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from django.contrib.auth import views as auth_views
 from . import views
 
 urlpatterns = [
     path('', views.index, name='default'),
     path('cases/', include('case_study.urls', namespace='cases')),
     path('user/', include('accounts.urls', namespace='accounts')),
     path('admin/', admin.site.urls),
+    
+    path('password_change', auth_views.PasswordChangeView.as_view(template_name='password_change.html'), name='password_change'),
```

Comment:
No “_” in paths. Should be 
“password/change”
“password/change/done”
“password/reset”
“password/reset/done”
“password/reset/confirm”
“password/reset/complete”

Reply:
Should these urls be in the accounts app’s URL file